### PR TITLE
NO-JIRA clarify console access using certs

### PR DESCRIPTION
**Pull request by brusdev**

### CODE REVIEW

File: docs/user-manual/en/security.md [hunk 1]
```diff
@@ -1323,6 +1323,32 @@ extra properties described as below.
 
 - `trustStorePassword` - The trust store's password.
 
+### Config access using client certificates
+The web console supports the authentication with client certificates, see the following steps:
```
Comment (pwright) [inline docs/user-manual/en/security.md:1327]:
```suggestion
The web console supports authentication with client certificates, see the following steps:  
```

File: docs/user-manual/en/security.md [hunk 2]
```diff
@@ -1323,6 +1323,32 @@ extra properties described as below.
...
+activemq-cert {
+   org.apache.activemq.artemis.spi.core.security.jaas.TextFileCertificateLoginModule required
+       debug=true
+       org.apache.activemq.jaas.textfiledn.user="cert-users.properties"
+       org.apache.activemq.jaas.textfiledn.role="cert-roles.properties";
+};
+```
+- Change the hawtio realm to match the realm defined in the `login.config` file
+for the [certificate login module](#certificateloginmodule). This is configured in the `artemis.profile` via the system property `-Dhawtio.role=activemq-cert`.
+- Enable secure access using HTTPS protocol with the client authentication:
```
Comment (pwright) [inline docs/user-manual/en/security.md:1339]:
```suggestion
- Enable secure access using HTTPS protocol with client authentication:
```

File: docs/user-manual/en/security.md [hunk 3]
```diff
@@ -1323,6 +1323,32 @@ extra properties described as below.
...
+    path="web"
+    keyStorePath="${artemis.instance}/etc/keystore.jks"
+    keyStorePassword="password"
+    clientAuth="true"
+    trustStorePath="${artemis.instance}/etc/truststore.jks"
+    trustStorePassword="password">
+    <app url="jolokia" war="jolokia-war-1.3.5.war"/>
+</web>
+```
+
```
Comment (pwright) [inline docs/user-manual/en/security.md:1351]: where is the client cert?

Comment (brusdev) [reply docs/user-manual/en/security.md:1351]: the client cert with the public key should be included in the `${artemis.instance}/etc/truststore.jks` (trustStorePath) while the client cert with the private key should be used by the client to set up the secure channel, i.e. if the client app is a browser the user should install the client cert with the private key in the browser, let me clarify it

**Review by pwright — APPROVED** [review]
looks good, I don't have privs to resolve previous conversations

**Comment by brusdev** [comment]
@pwright thanks for your review

**Pull request merged**